sources/packages/api/src-sql/Restrictions/Clean-restriction-add-action-and-notification-type-remove-allowed-count-and-seed.sql

```diff
@@ -0,0 +1,1326 @@
+    ARRAY ['No effect'] :: sims.restriction_action_types [],
+    'No effect'
+  );
+
+INSERT INTO
+  sims.restrictions(
+    restriction_type,
+    restriction_code,
+    description,
+    restriction_category,
+    action_type,
+    notification_type
+  )
+VALUES
+  (
+    'Federal',
+    'O',
+    'Please see overaward page for more info.',
+    'Federal',
+    ARRAY ['Stop disbursement'] :: sims.restriction_action_types [],
+    'Error'
+  );
+
+INSERT INTO
+  sims.restrictions(
+    restriction_type,
+    restriction_code,
+    description,
+    restriction_category,
+    action_type,
+    notification_type
+  )
+VALUES
+  (
+    'Provincial',
+    'OVRD',
+    'We have overriden the disbursement and this stays until the new amount goes out.',
+    'Other',
+    ARRAY ['No effect'] :: sims.restriction_action_types [],
+    'No effect'
+  );
+
+INSERT INTO
+  sims.restrictions(
+    restriction_type,
+    restriction_code,
+    description,
+    restriction_category,
+    action_type,
+    notification_type
+  )
+VALUES
+  (
+    'Federal',
+    'P',
+    'Convicted of an Offense related to Canada Student Loans.',
+    'Federal',
+    ARRAY ['Stop disbursement'] :: sims.restriction_action_types [],
+    'Error'
+  );
+
+INSERT INTO
+  sims.restrictions(
+    restriction_type,
+    restriction_code,
+    description,
+    restriction_category,
+    action_type,
+    notification_type
+  )
+VALUES
+  (
+    'Federal',
+    'R',
+    'Signifies that a Financial Institution (FI) has sent back a risk-shared loan to the federal government for further collection.',
+    'Federal',
+    ARRAY ['Stop disbursement'] :: sims.restriction_action_types [],
+    'Error'
+  );
+
+INSERT INTO
+  sims.restrictions(
+    restriction_type,
+    restriction_code,
+    description,
+    restriction_category,
+    action_type,
+    notification_type
+  )
+VALUES
+  (
+    'Federal',
+    'RB',
+    'Bankruptcy has been filed and there is a hold on federal and BC funding.',
+    'Federal',
+    ARRAY ['Stop disbursement'] :: sims.restriction_action_types [],
+    'Error'
+  );
+
+INSERT INTO
+  sims.restrictions(
+    restriction_type,
+    restriction_code,
+    description,
+    restriction_category,
+    action_type,
+    notification_type
+  )
+VALUES
+  (
+    'Federal',
+    'RJ',
+    'A judgement has been registered by CSFAP and student is not eligible for student loans.',
+    'Federal',
+    ARRAY ['Stop disbursement'] :: sims.restriction_action_types [],
+    'Error'
+  );
+
+INSERT INTO
+  sims.restrictions(
+    restriction_type,
+    restriction_code,
+    description,
+    restriction_category,
+    action_type,
+    notification_type
+  )
+VALUES
+  (
+    'Federal',
+    'RP',
+    'Student has been convicted of an offense and is not eligible for student loans.',
+    'Federal',
+    ARRAY ['Stop disbursement'] :: sims.restriction_action_types [],
+    'Error'
+  );
+
+-- TODO: Action type enum does not have 'Stop full time', need to confirm.
```

Comment:
@ann-aot it is probably the case that the enum is not updated. As per the upcoming ticket, we have both.
https://app.zenhub.com/workspaces/student-information-management-system-5fce9df5aa1b45000e937014/issues/bcgov/sims/1197
![image](https://user-images.githubusercontent.com/61259237/168858921-5e59b641-4d96-4a88-8762-786d87dd68c8.png)

Reply:
mm... okay.., my story these are the action type enum, 
![image](https://user-images.githubusercontent.com/77353155/168870804-8b7dbb9d-dbd3-40aa-8156-6f8b84355a01.png)